editorial/content-catalog.html

```
@@ -0,0 +1,519 @@
+<title>Namefi 内容目录</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)